src/utils.ts

```diff
@@ -0,0 +1,24 @@
+import * as execa from "execa";
+
+export const createExecaCommand =
+  (command: string, config: {}) => async (params: string[]) =>
+    execa(command, params, config);
+
+export const parseGitOutput = (str?: string) => {
+  if (!str) return "";
+
+  const gitOutRegex = /^ (?<flag>[\!\+\*\=t]) (\[(?<summary>[a-z ]+)\])?/im;
+  const match = str.match(gitOutRegex);
+  if (!match || !match.groups) return "";
+
+  return match.groups.flag;
```

Comment:
might want to return groups instead

Reply:
I renamed the method to specify that it returned just the flag. Hope that's fine.